guides/16_Video_Layout.md

```diff
@@ -0,0 +1,527 @@
+# Managing Video Quality for Different Video Layouts
+
+## Overview
+
+The Amazon Chime SDK for JavaScript enables builders to create applications with custom and dynamic video layouts. Depending on the number of video tiles displayed on the screen, their relative display size and priority, different video upstream strategies and tile management methods are applied to provide the optimal video experience for end users.
+
+The first section introduces the basic concepts, including video lifecycle management APIs and the uplink and downlink policies. The second section discusses several common use cases and how to implement them using the [Amazon Chime SDK for JavaScript](https://github.com/aws/amazon-chime-sdk-js) (“JavaScript SDK”) and the [Amazon Chime SDK React Components Library](https://github.com/aws/amazon-chime-sdk-component-library-react) (“React Components Library”).
+
+## Introduction
+
+### Video Lifecycle Management APIs
+
+Let us consider a scenario where a meeting has three attendees: attendee 0, attendee 1, and attendee 2. They are all sharing their local camera stream in a meeting, and attendee 2 is additionally sharing their screen via content share. The diagram below illustrates how all the video streams are received and managed from the perspective of attendee 0.
+
+![video lifecycle diagram](media://video_lifecycle.png)
+
+Typically, there are four steps involved in the lifecycle of a remote video with respect to a single receiving attendee:
+
+1. As remote attendees change their video stream, an up to date list of video sources (ignoring the receiving attendees’ own local video) is provided via the `remoteVideoSourcesDidChange` callback.
+2. The receiving client will select the remote video sources (and possibly which simulcast layer) that they wish to receive, which will initiate a negotiation (or “subscription”) with the backend servers to start forwarding those videos, depending on a client’s downlink policy (explained in a later section).
```

Comment:
I don't understand the meaning in the brackets: (and possibly which simulcast layer). Do you mean which simulcast layer?